package.json

```diff
@@ -5,10 +5,7 @@
   "main": "terrabrasilis.js",
   "scripts": {
     "test": "./node_modules/mocha/bin/mocha",
-    "test:watch": "./node_modules/mocha/bin/mocha --watch",
-    "preversion": "npm test",
-    "version": "git add .",
-    "postversion": "git push && git push --tags && npm publish"
```

Comment:
These lines is used to versioning using semantic versioning and npm.

Reply:
we're not publishing it on NPM anymore, all the dependencies on other repositories are pointing directly to GITHUB